package.json

```diff
@@ -208,6 +208,7 @@
     "@vercel/ncc": "^0.27.0",
     "@welldone-software/why-did-you-render": "7.0.1",
     "ajv-cli": "^5.0.0",
+    "axios": "^1.4.0",
```

Comment:
ah I didn't know `fetch` exits in node run time. trying to see if fetch works

Reply:
```
hayatasuenaga@Hayatas-Work-MBP App % npm run configure-mapbox

> new.expensify@1.3.55-7 configure-mapbox
> ts-node scripts/setup-mapbox-sdk-local.ts

Error: fetch is not defined
```

Fetch API was apparently introduced in Node 17. App repo somehow still uses node 16 (see the `engines` field in `package.json`) 😭 so `fetch` is not available.

_ts-node uses the node that is active in the current terminal session, which is node 16 in the App repo_

Reply:
The other option is to use `https` module from node but the syntax is kind of verbose (even considering that we only need to make one API call). So, I'm thinking of sticking with `axios`. What do you think @allroundexperts

Reply:
Since there is only one API call, I think using `https` is fine. Yes, it's more verbose, but adding this package just to make one API call is overkill.